terra2/assetlist.json

```diff
@@ -58,6 +58,114 @@
         }
       ]
     },
+    {
+      "description": "Dinheiros is the reserve currency of dioalma.protocol, an undivisible and rare token.",
+      "denom_units": [
+        {
+          "denom": "cw20:terra1spkm49wd9dqkranhrks4cupecl3rtgeqqljq3qrvrrts2ev2gw6sy5vz3k",
+          "exponent": 0
+        },
+        {
+          "denom": "Dinheiros",
+          "exponent": 0
+        }
+      ],
+      "type_asset": "cw20",
+      "address": "terra1spkm49wd9dqkranhrks4cupecl3rtgeqqljq3qrvrrts2ev2gw6sy5vz3k",
+      "base": "cw20:terra1spkm49wd9dqkranhrks4cupecl3rtgeqqljq3qrvrrts2ev2gw6sy5vz3k",
+      "name": "dinheiro",
+      "display": "Dinheiros",
+      "symbol": "Dinheiros",
+      "logo_URIs": {
+        "png": "https://static.wixstatic.com/media/1e62a0_04207897171e4f5c8931295172b3e34c~mv2.png"
```

Comment:
Please upload the images for addition here (into the /images/ folder) and point these URLs to the corresponding Chain Registry location. e.g., 'https://github.com/cosmos/chain-registry/terra2/images/dinheiros.png'

Reply:
done. thank you.